README.adoc

```diff
@@ -1,97 +1,114 @@
 
 [source,xml]
 ----
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+  <extension>
+    <groupId>org.l2x6.maven.srcdeps</groupId>
+    <artifactId>srcdeps-maven-local-repository</artifactId>
+    <version>0.0.12-SNAPSHOT</version>
+  </extension>
+</extensions>
+----
+
+==== 2. `.mvn/srcdeps.yaml`
 
-  [...]
-
-  <dependencies>
-    <dependency>
-      <groupId>org.l2x6.maven.srcdeps.itest</groupId>
-      <artifactId>srcdeps-test-artifact</artifactId>
-      <!-- -SRC- is the marker that makes Srcdeps Plugin to interpret the rest of the version
-           string as a scmVersionType, scmVersion pair. In this case scmVersionType is
-           "revision" and scmVersion is "66ea95d890531f4eaaa5aa04a9b1c69b409dcd0b" - i.e.
-           the sha1 of the git commit we want to build as a dependency.
-           scmVersionType values are given by the SCM (Source Code Management) system in use.
-           For git, scmVersionType values are "revision", "tag" or "branch"
-      -->
-      <version>0.0.1-SRC-revision-66ea95d890531f4eaaa5aa04a9b1c69b409dcd0b</version>
-    </dependency>
-  </dependencies>
-
-  <build>
-    <plugins>
-     [...]
-
-      <plugin>
-        <groupId>org.l2x6.maven.srcdeps</groupId>
-        <artifactId>srcdeps-maven-plugin</artifactId>
-        <version>0.0.1</version>
-        <extensions>true</extensions>
-        <configuration>
-          <repositories>
-            <repository>
-              <!-- a build tree unique id chosen by you -->
-              <id>org.hawkular.cmdgw</id>
-              <selectors>
-                <!-- groupId of the dependencies that come from this repo
-                     The syntax may change in the future,
-                     see https://github.com/l2x6/srcdeps-maven-plugin/issues/10 -->
-                <selector>org.hawkular.cmdgw</selector>
-              </selectors>
-              <!-- The url that is passed to checkout mojo of SCM Plugin,
-                   see https://maven.apache.org/scm/maven-scm-plugin/checkout-mojo.html -->
-              <url>scm:git:https://github.com/hawkular/hawkular-command-gateway.git</url>
-            </repository>
-          </repositories>
-        </configuration>
-      </plugin>
-
-    </plugins>
-  </build>
-
-  [...]
+ Various settings have to be configured in `.mvn/srcdeps.yaml` file. Especially, the mapping from groupId's of
+ dependencies to SCM repositiries where their sources reside.
 
+[source,yaml]
 ----
+configModelVersion: 1.0
+forwardProperties:
+- myForwardProperty
+builderIo:
+  stdin: inherit
+  stdout: inherit
+  stderr: inherit
+skip: false
+skipTests: false
+sourcesDirectory: ~/.m2/srcdeps
+verbosity: info
+repositories:
+  hawkularCommons:
+    selectors:
+    - org.hawkular.commons # if one of the selectors matches the groupId of a -SRC- dependency,
+                           # then the urls and buildArguments are used to build that dependency
```

Comment:
> What are selectors? Is this a term that is relevant for the implementation, or is it something I should know as a consumer of srcdeps? 

Consumers need to set them up. Is the new docs clear enough? https://github.com/ppalaga/srcdeps/commit/ca1f5d5989c41df7a8a6f618ee0f1d3e101aa3f4#diff-a1d5ae3027c5ac746a74e8814bad2274R39

> I'd change the term to "groupIds"

Yes, ATM they are just groupIds, but maybe one day we'll need to make them more pattern-like, something like: `org.hawkular.*` or `org.hawkular.commons:my-artifact`, etc. and for that approach, the selector term still suits better.